go/vt/vtgate/engine/delete.go

```diff
@@ -250,6 +190,42 @@ func (del *Delete) deleteVindexEntries(vcursor VCursor, bindVars map[string]*que
 	return nil
 }
 
+func (del *Delete) deleteVindexEntriesScatter(vcursor VCursor, bindVars map[string]*querypb.BindVariable, rss []*srvtopo.ResolvedShard) error {
+	queries := make([]*querypb.BoundQuery, len(rss))
+	for i := range rss {
+		queries[i] = &querypb.BoundQuery{Sql: del.OwnedVindexQuery, BindVariables: bindVars}
+	}
+	subQueryResults, errors := vcursor.ExecuteMultiShard(rss, queries, false, false)
+	for _, err := range errors {
+		if err != nil {
+			return vterrors.Wrap(err, "deleteVindexEntriesScatter")
+		}
+	}
+
+	if len(subQueryResults.Rows) == 0 {
+		return nil
+	}
+
+	for _, row := range subQueryResults.Rows {
+		ksid := row[0].Raw()
```

Comment:
This should be `ToBytes`. For all practical purposes, they are functionally the same. But `Raw` should only be used for encoding. Same change needed for update.